core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/documentation/utils.kt

```diff
@@ -19,24 +19,31 @@ import kotlin.annotation.AnnotationTarget.VALUE_PARAMETER
  * {@include [Indent]}
  *
  */
+@ExcludeFromSources
```

Comment:
These documentation interfaces can be [excluded](https://github.com/Kotlin/dataframe/blob/master/KDOC_PREPROCESSING.md#excludefromsources-annotation-excluding-code-content-from-sources) from the sources as they are only ever accessed from kdoc.